src/vs/code/electron-browser/proxy/auth.html

```diff
@@ -18,7 +18,7 @@
 		}
 
 		body {
-			font-family: system-ui, -apple-system, BlinkMacSystemFont, "Segoe WPC", "Segoe UI", "Ubuntu", "Droid Sans", sans-serif;
+			font-family: -apple-system, BlinkMacSystemFont, "Segoe WPC", "Segoe UI", system-ui, "Ubuntu", "Droid Sans", sans-serif;
```

Comment:
@joaomoreno @chrmarti note that `auth.html` does not seem to have a notion of platform, I think it should follow the same structure as process explorer and issue reporter.